### PR TITLE
ui: set max brightness when showing the QR code

### LINKED ISSUE #28906: ui: screen brightness at 100% while pairing QR code is up

### PR DESCRIPTION
**Pull request by deanlee**
resolve: [#28906](https://github.com/commaai/openpilot/issues/28906)

### CODE REVIEW

File: selfdrive/ui/qt/widgets/prime.cc [hunk 1]
```diff
@@ -26,10 +26,13 @@ PairingQRWidget::PairingQRWidget(QWidget* parent) : QWidget(parent) {
 void PairingQRWidget::showEvent(QShowEvent *event) {
   refresh();
   timer->start(5 * 60 * 1000);
+  // set max brightness
+  Hardware::set_brightness(100);
```
Comment (adeebshihadeh) [inline selfdrive/ui/qt/widgets/prime.cc:30]: think we want to only call this at one place in the ui. can you make some intermediate state that tells the Device class we want 100% brightness?

**Comment by deanlee** [comment]
how about this one?

**Comment by deanlee** [comment]
convert to draft. we can use device() to access the device class after https://github.com/commaai/openpilot/pull/28904

**Comment by sshane** [comment]
it still goes through the filter, so it takes a while to ramp up and down from max brightness. we probably want it instantly here, right @adeebshihadeh?

**Comment by adeebshihadeh** [comment]
I just remembered we won't even get full brightness anyway because of [brightnessd](https://github.com/commaai/agnos-builder/blob/master/userspace/usr/comma/brightnessd.py). I'll have to think about this one some more.

**Comment by adeebshihadeh** [comment]
> it still goes through the filter, so it takes a while to ramp up and down from max brightness. we probably want it instantly here, right @adeebshihadeh?

This should be fine.

**Pull request merged**